src/components/inputs/RadioButton/RadioButton.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import webexComponentClasses from '../../helpers';
+
+/**
+ * RadioButton component.
+ *
+ * @param {object} props  React props passed to the component
+ * @param {string} [props.className]  Custom CSS class to apply
+ * @param {string} props.name  Radio button name
+ * @param {string} props.value  Radio button value
+ * @param {boolean} [props.selected=false]  Flag indicating whether radio button is selected
+ * @param {boolean} [props.disabled=false]  Flag indicating whether radio button is disabled
+ * @param {boolean} [props.required=false]  Flag indicating whether radio button is required
+ * @param {string} [props.error]  Error text
+ * @param {number} props.tabIndex  Value of the tabIndex
+ * @param {Function} [props.onChange]  Action to perform on radio button change
+ * @returns {object} JSX of the component
+ */
+
+export default function RadioButton({
+  className,
+  name,
+  value,
+  selected,
+  disabled,
+  required,
+  error,
+  onChange,
+  tabIndex,
+}) {
+  const enabled = !disabled;
+  const SPACE_KEY = ' ';
+  const ENTER_KEY = 'Enter';
+
+  const [cssClasses] = webexComponentClasses('radio-button', className, {
+    enabled,
+    disabled,
+    error,
+  });
+
+  return (
+    <div
```

Comment:
still wondering if this container is really needed. Can we move all its props on the `<input type="radio">` element? This way, the role/aria attributes won't be needed anymore. I think not even the keyboard handling, as the radio button natively has that.